src/main/java/seedu/address/Help.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address;
```

Comment:
For the actual TrAcker, there is already an existing HelpWindow in seedu.address.ui. So maybe we can customise from there instead of creating from scratch.